ocaml/xapi/xapi_globs.ml

```diff
@@ -1388,6 +1392,11 @@ let other_options =
     , (fun () -> string_of_bool !ignore_vtpm_unimplemented)
     , "Do not raise errors on use-cases where VTPM codepaths are not finished."
     )
+  ; ( "override-uefi-certs"
+    , Arg.Set override_uefi_certs
+    , (fun () -> string_of_bool !override_uefi_certs)
+    , "Enable/Disable overriding location for varstored UEFI certificates"
+    )
```

Comment:
Is there a reason `varsore_dir` & `default_auth_dir` are not configurable?

Reply:
Why would they be? one is defined by the rpm, and the other is an interface with varstored an xapi, neither of which make sense to allow for users to configure

Reply:
No it's fine by me, i was just wondering ^^.